documents/motions/10.2024.4 - Appeals Committee.md

```diff
@@ -0,0 +1,33 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 10.2024.4  
+**Subject:** WCA Appeals Committee  
+**Intent:** Rights and duties of the WCA Appeals Committee  
+**Submitted by:** WCA Board  
+**Date:** TBC, 2024  
+
+# Motion
+
+The WCA Appeals Committee is an Advisory Committee of the WCA.
+
+1. The WCA Appeals Committee has the following rights and duties:
+   1. Forming the Board of Appeal to carry out the appeals processes described in [WCA Motion : Disputes](wca{documents/motions/16.2021.1 - Disputes.pdf}).
```

Comment:
I think there needs to be something here that acknowledges that the Board of Appeal is a temporary structure that exists just for the life of this specific appeal.

Reply:
Suggested wording "Forming a Board of Appeal for a dispute.  The Board of Appeal is a temporary team to carry out the appeals processes described in ..."

Reply:
I have incorporated changes for this.